docs/pages/faq.mdx

```diff
@@ -149,6 +149,8 @@ Teleport provides security-critical support for the current and two previous rel
 
 | Release | Long Term Support | Release Date | Min tsh version |
 | - | - | - | - |
+| 6.2 | Yes | May 21th, 2021 | 3.1 |
+| 6.1 | Yes | April 9th, 2021 | 3.1 |
```

Comment:
I don't think that 6.1 and 6.1 are long term support, so let's say No in that section